packages/blade/src/components/Tour/Tour.web.tsx

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable consistent-return */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import React, { useState, useCallback, useMemo } from 'react';
+import { FloatingPortal } from '@floating-ui/react';
+import { TourContext } from './TourContext';
+import { TourPopover } from './TourPopover';
+import {
+  smoothScroll,
+  useDelayedState,
+  useIntersectionObserver,
+  useIsTransitioningBetweenSteps,
+  useLockBodyScroll,
```

Comment:
Modal already does this via https://floating-ui.com/docs/floatingoverlay#lockscroll 

We need to do this ourselves because we are not using FloatingOverlay (since we have that svg mask)